index.bs

```diff
@@ -1509,6 +1513,9 @@ during registration.
         [=attestation statement=], as well as to decode and validate the [=authenticator data=] along with the
         [=JSON-serialized client data=]. For more details, see [[#sctn-attestation]], [[#generating-an-attestation-object]],
         and [Figure 3](#fig-attStructs).
+
+    :   <dfn>transport</dfn>
+    ::  This attribute indicates which [transport](#transport) the client used to communicate with the authenticator.
```

Comment:
suggest using [=client=] and [=authenticator=] here.

Reply:
Done.